crates/ox_content_ssg/src/html/reader_chrome.css

```
@@ -1,13 +1,11 @@
 .ox-code {
-  --ox-copy-reserved-inline-size: 4.75rem;
+  --ox-copy-reserved-inline-size: 2.5rem;
   position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)